platform/ios/src/MGLMapAccessibilityElement.mm

```diff
@@ -52,6 +52,9 @@ - (instancetype)initWithAccessibilityContainer:(id)container feature:(id<MGLFeat
         NSString *nameAttribute = [NSString stringWithFormat:@"name_%@", languageCode];
         NSString *name = [feature attributeForKey:nameAttribute];
 
+        if (name == nil && [feature attributeForKey:@"name"] != nil) {
+            name = [feature attributeForKey:@"name"];
+        }
```

Comment:
This fallback should probably go further down, after the transliteration step. The transliteration step depends on knowing the language the name is in, but we don’t know the language of the language-agnostic `name` property.

Reply:
On second thought, Streets source v8 comes with a [`name_script`](https://docs.mapbox.com/vector-tiles/reference/mapbox-streets-v8/#name_script-text) property that we can feed into `-mgl_stringByTransliteratingIntoScript:` in the event that we don’t know the language. So call it with the language code for `name_*` but with `name_script` for `name`.

Reply:
It doesn't look like the values for `name_script` match the values `-mgl_stringByTransliteratingIntoScript:` expects. For example, `name_script` returns Latin, while `-mgl_stringByTransliteratingIntoScript:` expects `Latn`. I'll move the check for `name` a little further down.

Reply:
`-[NSString stringByApplyingTransform:]` expects an [ICU transliterator identifier](http://userguide.icu-project.org/transforms/general#TOC-Transliterator-Identifiers). `Latin` is a valid identifier – probably most of those `name_script` values are valid identifiers, so it would be fine to dump it into the method and see what comes out.